vnext/Chakra/ChakraHelpers.cpp

```diff
@@ -319,6 +199,57 @@ void ChakraJSException::buildMessage(JsValueRef exn, JsValueRef sourceURL, const
   }
 }
 
+bool ChakraVersionInfo::initialize() noexcept {
+#if defined(CHAKRACORE_UWP)
+  // This code is win32 only at the moment. We will need to change this
+  // line if we want to support UWP.
+  return false;
```

Comment:
>false [](start = 9, length = 5)

This used to say that it succeeded. Changed that here, since it would break if we used this on UWP.